### ISSUE TITLE
Create resilience4j module for feign

### ISSUE THREAD

**Issue by pwlan** [resilience4j/resilience4j#196]
Hi,

the idea is to create a new module for the feign framework (https://github.com/OpenFeign/feign).
It should be similar to resilience4j-retrofit or https://github.com/OpenFeign/feign/tree/master/hystrix.

**Comment by RobWin** [comment]
Good idea :)

**Comment by storozhukBM** [comment]
@pwlan would you like to contribute and maintain such module?

**Comment by pwlan** [comment]
Sure, i thought you would never ask ;) I will go through some design ideas with RobWin this week and hopefully start a PR soon.

**Comment by typekpb** [comment]
any updates on this one? the contribution would be probably done in the https://github.com/OpenFeign/feign right?

**Comment by typekpb** [comment]
OK, just noticed, it has been already done :) ignore my last comment, please

**Comment by typekpb** [comment]
Started to play around with the implementation. Currently blocked by the missing `resilience4j-feign` in the snapshot repo: http://oss.jfrog.org/oss-snapshot-local/io/github/resilience4j/

@storozhukBM Is there a chance to change the project version to whatever you plan to release next (probably `0.14.0-SNAPSHOT`) and push to upstream snapshots maven repo?

**Comment by typekpb** [comment]
Sure, thanks a lot!

**Comment by storozhukBM** [comment]
@typekpb Snapshot deployed. Have a nice day)

**Comment by typekpb** [comment]
Thanks!

**Comment by awalland** [comment]
is there any timeframe when a non-snapshot artifact of this module will be released? i'd love to use it but i can't use snapshot versions.

thanks!

**Comment by dharmeshjogadia** [comment]
Hi @storozhukBM

When can we expect `0.14.0-SNAPSHOT` release? I'd like to experiment with my services.
thanks.
